treewide: Modify the `wifi_cred add` command in examples

rado17 opened the pull request:
`wifi_cred add` is modified now. Update the examples in samples and documentation to reflect the same.

NordicBuilder commented:
## CI Information

<sup>_To view the history of this post, clich the 'edited' button above_</sup>
Build number: 1
### Inputs:
#### Sources:
sdk-nrf: PR head: [3a9950bfe77d8ae2a796c68b4cb1f0e95a6b1174](https://github.com/nrfconnect/sdk-nrf/pull/18272/commits/3a9950bfe77d8ae2a796c68b4cb1f0e95a6b1174)
<details><summary> more details</summary>

sdk-nrf:
> PR head: [3a9950bfe77d8ae2a796c68b4cb1f0e95a6b1174](https://github.com/nrfconnect/sdk-nrf/pull/18272/commits/3a9950bfe77d8ae2a796c68b4cb1f0e95a6b1174)
> merge base: [d3a8699e25991a8f624603da278272728c913657](https://github.com/nrfconnect/sdk-nrf/commit/d3a8699e25991a8f624603da278272728c913657)
> target head (main): [d3a8699e25991a8f624603da278272728c913657](https://github.com/nrfconnect/sdk-nrf/commit/d3a8699e25991a8f624603da278272728c913657)
> [Diff](https://github.com/nrfconnect/sdk-nrf/compare/d3a8699e25991a8f624603da278272728c913657..3a9950bfe77d8ae2a796c68b4cb1f0e95a6b1174)</details>

#### Github labels
| Enabled | Name | Description |
| -- | -- | -- |
|| ci-disabled | _Disable the ci execution_ | 
|| ci-all-test | _Run all of ci, no test spec filtering will be done_ | 
|| ci-force-downstream | _Force execution of downstream even if twister fails_ | 
|| ci-run-twister | _Force run twister_ | 
|| ci-run-zephyr-twister | _Force run zephyr twister_ | 

<details><summary> List of changed files detected by CI (3)</summary>

```
doc
│  ├── nrf
│  │  ├── includes
│  │  │  │ wifi_credentials_shell.txt
samples
│  ├── cellular
│  │  ├── nrf_cloud_multi_service
│  │  │  │ README.rst
│  ├── net
│  │  ├── http_server
│  │  │  │ README.rst
```
</details>

### Outputs:

#### Toolchain
Version: 
Build docker image: 

#### Test Spec & Results: :white_check_mark: Success; :x: Failure; :orange_circle: Queued; :yellow_circle: Progress; :white_medium_square: Skipped; :warning: Quarantine
- :orange_circle: Toolchain
- :orange_circle: Build twister
- :orange_circle: Integration tests
   - :orange_circle: test-fw-nrfconnect-nrf-iot_cloud

<details><summary>Disabled integration tests</summary>

- 
   - desktop52_verification
   - doc-internal
   - test_ble_nrf_config
   - test-fw-nrfconnect-apps
   - test-fw-nrfconnect-ble_mesh
   - test-fw-nrfconnect-ble_samples
   - test-fw-nrfconnect-boot
   - test-fw-nrfconnect-chip
   - test-fw-nrfconnect-fem
   - test-fw-nrfconnect-nfc
   - test-fw-nrfconnect-nrf-iot_libmodem-nrf
   - test-fw-nrfconnect-nrf-iot_lwm2m
   - test-fw-nrfconnect-nrf-iot_mosh
   - test-fw-nrfconnect-nrf-iot_nrf_provisioning
   - test-fw-nrfconnect-nrf-iot_positioning
   - test-fw-nrfconnect-nrf-iot_samples
   - test-fw-nrfconnect-nrf-iot_thingy91
   - test-fw-nrfconnect-nrf-iot_zephyr_lwm2m
   - test-fw-nrfconnect-nrf_crypto
   - test-fw-nrfconnect-proprietary_esb
   - test-fw-nrfconnect-rpc
   - test-fw-nrfconnect-rs
   - test-fw-nrfconnect-tfm
   - test-fw-nrfconnect-thread
   - test-fw-nrfconnect-zigbee
   - test-low-level
   - test-sdk-audio
   - test-sdk-dfu
   - test-sdk-find-my
   - test-sdk-mcuboot
   - test-sdk-pmic-samples
   - test-sdk-sidewalk
   - test-sdk-wifi
</details>



*Note: This message is automatically posted and updated by the CI*

peknis commented on doc/nrf/includes/wifi_credentials_shell.txt:
```diff
@@ -9,9 +9,10 @@ Once you have flashed your device with this sample, connect to your device's UAR
+   wifi_cred add -s *NetworkSSID* -k *SecurityMode* -p *NetworkPassword*
 
-Where *NetworkSSID* is replaced with the SSID of the Wi-Fi access point you want your device to connect to, and *NetworkPassword* is its password.
+Where *NetworkSSID* is replaced with the SSID of the Wi-Fi access point you want your device to connect to, and *NetworkPassword* is its password. *SecurityMode* is replaced by the number as listed here:
+ 0:None, 1:WPA2-PSK, 2:WPA2-PSK-256, 3:SAE-HNP, 4:SAE-H2E, 5:SAE-AUTO, 6:WAPI, 7:EAP-TLS, 8:WEP, 9: WPA-PSK, 10: WPA-Auto-Personal, 11: DPP.
```
```suggestion
Where *NetworkSSID* is replaced with the SSID of the Wi-Fi access point you want your device to connect to, and *NetworkPassword* is its password.
*SecurityMode* is replaced by the number as listed here:

* 0:None
* 1:WPA2-PSK
* 2:WPA2-PSK-256
* 3:SAE-HNP
* 4:SAE-H2E
* 5:SAE-AUTO
* 6:WAPI
* 7:EAP-TLS
* 8:WEP
* 9: WPA-PSK
* 10: WPA-Auto-Personal
* 11: DPP
```

rado17 replied:
Done.

krish2718 reviewed (commented):
I can still few paths not updated:
```
nrf/samples/wifi/shell/README.rst
nrf/samples/cellular/nrf_cloud_multi_service/README.rst
```

NordicBuilder commented:
You can find the documentation preview for this PR at this <a href='http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-18272
'>link</a>. It will be updated about 10 minutes after the documentation build succeeds.

*Note: This comment is automatically posted by the Documentation Publishing GitHub Action.*

rado17 commented:
> I can still few paths not updated:
> 
> ```
> nrf/samples/wifi/shell/README.rst
> nrf/samples/cellular/nrf_cloud_multi_service/README.rst
> ```

Updated the complete `help` in shell sample. Updated the example in `nrf_cloud_multi_service`

peknis commented on doc/nrf/includes/wifi_credentials_shell.txt:
```diff
@@ -9,9 +9,24 @@ Once you have flashed your device with this sample, connect to your device's UAR
+* 6:WAPI
+* 7:EAP-TLS
+* 8:WEP
+* 9: WPA-PSK
+* 10: WPA-Auto-Personal
+* 11: DPP
```
```suggestion
* 9:WPA-PSK
* 10:WPA-Auto-Personal
* 11:DPP
```

rado17 replied:
Done.
